README.adoc

```diff
@@ -303,16 +342,18 @@ management.server-org.springframework.boot.actuate.autoconfigure.web.server.Mana
 ----
 ====
 
-The actuator exposes the following:
+By default, the actuator exposes the following endpoints:
 
-* http://localhost:8080/actuator/health[actuator/health]
-* http://localhost:8080/actuator[actuator]
+* http://localhost:8080/actuator/health[actuator/health] - Gives a summary of the health of the
+  application.
+* http://localhost:8080/actuator[actuator] - Lists all of the Actuator endpoints that are available
+in the application. You can configure your app the expose as many or
+few of the Actuator endpoints as you need.
 
 NOTE: There is also an `/actuator/shutdown` endpoint, but, by default, it is visible only
 through JMX. To http://docs.spring.io/spring-boot/docs/{spring_boot_version}/reference/htmlsingle/#production-ready-endpoints-enabling-endpoints[enable it as an HTTP endpoint], add
```

Comment:
In the latest version of Spring Boot JMX is also locked down so it's not exposed by default